setup.cfg

```diff
@@ -0,0 +1,48 @@
+[metadata]
+name = napari-console
+url = https://github.com/napari/napari-console
+license = BSD 3-Clause
+license_file = LICENSE
+description = A plugin that adds a console to napari
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Nicholas Sofroniew
+author_email = sofroniewn@gmail.com
```

Comment:
@sofroniewn I kept you, but we can mirror napari and make it the napari core devs.